actions/class.Category.php

```
@@ -59,6 +59,30 @@ public function getExposedsByClass()
+     *  { success : bool, data : {
+     *      propertyUri : { label, values: [invalidValue] }
+     *  } }
+     *
+     * @return void
+     */
+    public function getInvalidExposedCategories()
+    {
+        $id = $this->getRequestParameter('id');
+
+        if (is_null($id) || empty($id)) {
+            return $this->returnBadParameter('The item URI is required');
```

PHPStan (2.2.8)

[error] Method taoItems_actions_Category::getInvalidExposedCategories() with return type void returns null but should not return anything.


(return.void)